docs/shared/recipes/tips-n-tricks/migrating-to-flat-eslint.md

```diff
@@ -0,0 +1,25 @@
+# Switching to the ESLint's flat config format
+
+Version 8 of ESLint introduced a new configuration format called [Flat Config](https://eslint.org/docs/latest/use/configure/configuration-files-new). The purpose of this format is to:
+
+- propose a single configuration format (in contrast to the existing `JSON`, `Yaml` and `JS`-based configs)
```

Comment:
Instead of `propose`, I think you mean `create` or `provide`.  Its not a proposal

Reply:
I actually meant `propose` but I understand the confusion. I will change it.